kotlin-native/NEW_MM.md

```diff
@@ -104,8 +104,14 @@ Known issues:
 For the first preview, we're using the simplest scheme for garbage collection: single-threaded stop-the-world
 mark-and-sweep algorithm, which is triggered after enough functions, loop iterations, and allocations were executed. This greatly hinders the performance, and one of our top priorities now is addressing these performance issues.
 
-We don't have nice instruments to monitor the GC performance yet. So far, diagnosing requires looking at GC logs. To enable the logs, add the compilation flag `-Xruntime-logs=gc=info`. Or, in a Gradle build script:
+We don't have nice instruments to monitor the GC performance yet. So far, diagnosing requires looking at GC logs. To enable the logs, add the compilation flag `-Xruntime-logs=gc=info` using one of the following options:
 
+* In `gradle.properties`:
```

Comment:
Sorry, I wasn't clear. It's not yet possible to set up logging via `gradle.properties`.